src/procedures/procedures.11tydata.yaml

```diff
@@ -286,6 +286,186 @@ procedureMigrateZigbeeZHAOnly:
+          <li>Check if the ZHA integration shows up.</li>
+          <li>If the ZHA integration does not show up, select <b>Add integration</b>, then select <b>ZHA</b> from the list.</li>
+        </ol>
+    - title: Configure the new ZHA integration
+      image: conbee-migrate-zha-04.png
+      content: |
+        <ul>      
+          <li>Once the ZHA integration has detected ConBee II, select <b>Configure</b>.</li>
+        </ul>
+    - title: Set up ConBee II as ZHA integration
+      image: conbee-migrate-zha-05.png
+      content: |
+        <ul>
+          <li>Select <b>Submit</b>.</li>
+        </ul>
+    - title: Choose the network settings for the radio.
+      image: conbee-migrate-zha-06.png
+      content: |
+        <ul>
+          <li>Select <b>Keep radio network settings</b>.</li>
+        </ul>
+    - title: Wait
+      image: conbee-migrate-zha-07.png
+      content: |
+        <ul>
+          <li>Wait for the settings to be loaded.</li>
+        </ul>
+    - title: Add the Zigbee coordinator to your area
+      image: conbee-migrate-zha-09.png
+      content: |
+        <ol>
+          <li>Once deCONZ has been recognized as your Zigbee coordinator, add it to your area.</li>
+          <li>Select <b>Finish</b>.</li>
+        </ol>
+    - title: Wait for the devices rejoin the network.
+      image: conbee-migrate-zha-10.png
+      content: |
+        <ul>
+          <li class="info">Wait for the devices to rejoin the network. Normally, powered devices rejoin within one hour. You may be able to accelerate that process by power-cycling devices. Your battery-powered devices may need to be re-joined to the network manually.</li>
+          <li>Once the devices show up in the integration, select <b>Configure</b>.</li>
+        </ul>
+    - title: Migrate the network settings
+      image: conbee-migrate-zha-11.png
+      content: |
+        <ul>
+          <li>Under <b>Network Settings</b>, select <b>Migrate radio</b>.</li>
+        </ul>
+    - title: Reconfigure ZHA
+      image: conbee-migrate-zha-12.png
+      content: |
+        <ul>
+          <li>Select <b>Submit</b>.</li>
+        </ul>
+    - title: Migrate to a new radio
+      image: conbee-migrate-zha-13.png
+      content: |
+        <ul>
+          <li>Select <b>Migrate to a new radio</b>.</li>
+        </ul>
+    - title: Start the automatic backup
+      image: conbee-migrate-zha-14.png
+      content: |
+        <ol>
+          <li>Make sure the ConBee firmware is running firmware version 26720700.</li>
+          <li>When it is up-to-date, select <b>Submit</b>.</li>
+        </ol>
+    - title: Unplug your old radio
+      image: z2m-migrate-zha-06.png
+      content: |
+        <ol>
+          <li>If you no longer need the old radio stick, you can now remove it.</li>
+          <li>Before selecting <b>Submit</b>, connect SkyConnect, as described in the following step.</li>
```

Comment:
I like that we tell the user to plug in first already here, to prevent them from clicking submit too early  :+1: